src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -88,12 +99,53 @@
             }
         }
 
-        if (query.isPresent() && !StringUtils.isEmpty(query.get())) {
-            return actorRepository.findByTextSearchContainingOrderByActorId(query.get().toLowerCase());
-        } else {
-            // return the non-paged results
-            return actorRepository.findAll(Sort.by(Direction.ASC, "actorId"));
-        }
+        final FeedOptions options = new FeedOptions();
+        options.enableCrossPartitionQuery(true);
+        options.maxDegreeOfParallelism(2);
+
+        int skipCount = 0;
+        int takeCount = 10;
+
+        Flux<FeedResponse<CosmosItemProperties>> response = context.getBean(CosmosClient.class).getDatabase("imdb")
+                .getContainer("actors")
+                .queryItems("SELECT * from c OFFSET " + skipCount + " LIMIT " + takeCount, options);
+
```

Comment:
Need to move this logic into the ActorService or data access object.

Reply:
Agreed. See the comments with the Pull request, I acknowledge this. I will do this as part of #93 since it will require a decent bit of refactoring.

Reply:
This was done. Look at ActorsDao.java